File: codequest/2019SolutionsSG/Lockheed Solutions/Java/Prob04.java
/* CaughtSpeeding.java
 *
 * Copyright (c) 2019 Lockheed Martin (LM) as an unpublished work. 
 * All Rights Reserved.
 *
 * This file does not contain any Lockheed Martin Proprietary Information.
 */

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.FileInputStream;
import java.io.File;

public class Prob04 {

	public static void main(String[] args) throws Exception {
		try (BufferedReader input = new BufferedReader(new InputStreamReader(new FileInputStream(new File("Prob04.in.txt"))))) {
			int testCases = Integer.parseInt(input.readLine());

			for (int testcase = 0; testcase < testCases; testcase++) {
				String text = input.readLine();
				String[] data = text.split(" ");
				
				int speed = Integer.parseInt(data[0]);
				boolean birthday = Boolean.parseBoolean(data[1]);
				
				if(speed <= 60 || (birthday && speed <= 65)) {
					System.out.println("no ticket");
				}
				else if(speed <= 80 || (birthday && speed <= 85)) {
					System.out.println("small ticket");
				}
				else {
					System.out.println("big ticket");
				}
			}
		}
	}

}

File: codequest/2019Solutions/Lockheed Solutions/VB.NET/Prob04.vb
Option Explicit On
Module CaughtSpeeding

    Public Sub Main()
        Dim testCases As Integer
        Integer.TryParse(Console.ReadLine(), testCases)
        For testCase As Integer = 0 To (testCases - 1)
            solve()
        Next
    End Sub

    Sub solve()
        Dim LineArray = Console.ReadLine().Split(" ")
        Dim Speed = Convert.ToInt32(LineArray(0))
        Dim isBirthday = Convert.ToBoolean(LineArray(1))
        If Speed <= (60 + If(isBirthday, 5, 0)) Then
            Console.WriteLine("no ticket")
        ElseIf Speed <= (80 + If(isBirthday, 5, 0)) Then
            Console.WriteLine("small ticket")
        Else
            Console.WriteLine("big ticket")
        End If
    End Sub

End Module
